cfgmgr/sflowmgr.cpp

```diff
@@ -108,7 +109,8 @@ void SflowMgr::sflowUpdatePortInfo(Consumer &consumer)
             if (m_gEnable && m_intfAllConf)
             {
                 // If the Local Conf is already present, dont't override it even though the speed is changed
-                if (new_port || (speed_change && !m_sflowPortConfMap[key].local_conf))
+                if (new_port || (speed_change && 
+                   !(m_sflowPortConfMap[key].local_rate_cfg || m_sflowPortConfMap[key].local_admin_cfg)))
```

Comment:
This can be removed "|| m_sflowPortConfMap[key].local_admin_cfg" as the rate update based on speed change is only blocked if there is any local_rate_cfg. It doesn't depend on local_admin_cfg.

Reply:
ok